apps/explorer/lib/explorer/chain/smart_contract.ex

```diff
@@ -227,6 +232,8 @@ defmodule Explorer.Chain.SmartContract do
     field(:partially_verified, :boolean)
     field(:file_path, :string)
     field(:is_vyper_contract, :boolean)
+    field(:is_changed_bytecode, :boolean, default: false)
```

Comment:
i've seen this a few times in this PR, how does contract bytecode change after deployment? afaik it can only `selfdestruct` if that functionality was built into the contract

Reply:
scary